### DISCUSSION TITLE
海外服务器，播放视频，阿里云盘没问题，百度网盘有点小问题。

### DISCUSSION THREAD

**Discussion by davidtwchn** [AlistGo/alist#3767] [Q&A] [answered]
海外 linux 服务器，WebDAV 策略为“302 重定向”，播放百度网盘视频，提示“回放错误”
海外 linux 服务器，WebDAV 策略为“本地代理”，播放百度网盘视频，一切正常
国内 Windows 服务器，WebDAV 策略为“302 重定向”，播放百度网盘视频，一切正常。
问题：海外 linux 服务器百度网盘的 WebDAV 策略只能用”本地代理“吗？还是我哪个地方的设置错了。

============================================================
具体配置如下：
海外 linux 服务器，在圣何塞，CentOS 7
Alist 版本：3.13.1

国内服务器为腾讯服务器，在上海，Windows Server 2012 R2
Alist 版本：3.12.2

**Comment by davidtwchn** [comment, marked as answer]
收到电报群 Charles 回复：
可能是无解的，你海外 linux 服务器重定向指向的大概率是百度海外 cdn 服务器，极有可能禁止大陆 IP 访问，如果是这样，你海外就只能本地代理。

谢谢！:-)

**Comment by cnscorpions** [comment]
同样是海外的linux服务器，挂在百度云盘使用本地代理解决了我的问题，谢谢！
